docs/architecture/incremental.md

```diff
@@ -84,7 +84,17 @@ Some sources cannot define the cursor without user input. For example, in the [p
 
 \(You can find a more technical details about the configuration data model [here](catalog.md)\).
 
+## Getting the Latest Snapshot of data
+
+As demonstrated in the examples above, with **Incremental Append,** a record which was updated in the source will be appended to the destination rather than updated in-place. This means that if data in the source uses a primary key \(e.g: `user_id` in the `users` table\), then the destination will end up having multiple records with the same primary key value. 
+
+However, some use cases require only the latest snapshot of the data. If you want the latest snapshot and are syncing to a destination that supports views, we recommend creating a view on your data which groups by the primary key and deduplicates by the largest `_airbyte_emitted_at` values. The `_airbyte_emitted_at`  column is added by Airbyte to all records synced to the destination. 
```

Comment:
would it be valuable to write the query for an imaginary case? this is an annoying query to write. i think as a reader i would appreciate an example that i could copy and paste and the modify.

Reply:
Like it is done here?

https://docs.airbyte.io/tutorials/connecting-el-with-t-using-sql#customize-sql-query

Reply:
added a simple(r?) query here and also pointed to the docs